mappings/net/minecraft/recipe/Ingredient.mapping

```diff
@@ -29,3 +29,4 @@ CLASS bae net/minecraft/recipe/Ingredient
 	METHOD c toJson ()Lcom/google/gson/JsonElement;
 	METHOD d isEmpty ()Z
 	METHOD f createStackArray ()V
+	METHOD test includes (Ljava/lang/Object;)Z
```

Comment:
Looks inherited, I don't think this is supposed to be mapped.

Reply:
Definitely isn't.